gulpfile.js

```diff
@@ -40,10 +41,15 @@ let zaExtra7z = path.join(zaRoot, '7z920_extra.7z');
 let rhZip = path.join(zaRoot, 'resource_hacker.zip');
 let rhExe = path.join(zaRoot, 'ResourceHacker.exe');
 let zaElectronPackage = path.join(zaRoot, artifactName + '-win32-x64');
-let bundled7z = path.join(zaRoot, artifactName +'-win32-x64.7z');
-let installerExe = resolveInstallerExePath('');
-
-gulp.task('transpile:app', function() {
+let bundled7z = path.join(zaRoot, artifactName +'-win32-x64.7z');
+let installerExe = resolveInstallerExePath('');
+
+process.on('uncaughtException', function(err) {

```

Comment:
I'll remove this, I was looking into prefetch task problems and wanted to be sure there is are no errors in Promises.